xclim/sdba/adjustment.py

```diff
@@ -291,6 +295,10 @@ def adjust(
         xr.Dataset
             The bias-adjusted Dataset.
         """
+        if sim is None:
+            kwargs["sim_is_None"] = True
+            sim = hist.copy()
+
```

Comment:
I used here the same trick used before in `MultivariateAdjust`. That's really not ideal but as the classes are defined right now I think this is our best choice.

Reply:
Right, I think I have something a bit more elegant that will not require an extra input. This would mean changing the Adjust class so that `sim` is optional, and then:
```
# class Train
if sim is None:
   sim = hist.copy() 
   sim.attrs["_is_hist"] = True
...
# class DOTC
def _adjust(ref,hist, sim): 
   if "_is_hist" not in sim.attrs: 
       # throw an error
```